File: swift/iniciante/1009.swift
import Foundation

if let inputA = readLine(),
   let inputB = readLine(),
   let inputC = readLine(),
   let fixedSalary = Double(inputB),
   let totalSales = Double(inputC)
{
    let sellerName = inputA
    _ = sellerName // Ignora o aviso de que 'sellerName' não foi usada
    let commissionRate = 0.15
    let commission = totalSales * commissionRate
    let totalToReceive = fixedSalary + commission

    print(String(format: "TOTAL = R$ %.2f", totalToReceive))
}

File: typescript/iniciante/1009.ts
import { bold, yellow } from "https://deno.land/std@0.224.0/fmt/colors.ts";

const sellerName = prompt("Digite o nome do vendedor:")!; // não usado, mas mantido
const fixedSalary = Number(prompt("Digite o salário fixo:")!.replace(",", "."));
const totalSales = Number(
  prompt("Digite o total de vendas:")!.replace(",", "."),
);

const commissionRate = 0.15;
const commission = totalSales * commissionRate;
const totalToReceive = fixedSalary + commission;

console.log(yellow(`TOTAL = R$ ${bold(totalToReceive.toFixed(2))}`));
